mbed_lstools/lstools_base.py

```diff
@@ -366,12 +368,17 @@ def retarget_read(self):
                     str(e)))
         return {}
 
-    def retarget(self):
+    def retarget(self, enable=True):
```

Comment:
It seems funny to have a function that can optionally do nothing. Wouldn't you instead just want to optionally call the function?